indy_node/server/request_handlers/read_req_handlers/get_rich_schema_object_by_id_handler.py

```diff
@@ -12,6 +14,10 @@ def __init__(self, database_manager: DatabaseManager):
 
     def get_result(self, request: Request):
         self._validate_request_type(request)
+
+        if not getConfig().enableRichSchemas:
+            raise InvalidClientRequest(request.identifier, request.reqId, "RicheSchemas feature is disabled")
```

Comment:
I think there is a small typo here `RichESchemas`
We can make the message more detailed. Something like "RichSchema transaction is disabled" and "GetRichSchema query is disabled". But the current variant is ok.